api/envoy/config/core/v3/address.proto

```diff
@@ -134,15 +141,20 @@ message BindConfig {
   // precompiled binaries.
   repeated SocketOption socket_options = 3;
 
-  // Additional source addresses appended to the address specified in the `source_address`
-  // field. This enables to specify multiple source addresses. Currently, only one additional
-  // address can be supported, and the additional address should have a different IP version
+  // Extra source addresses appended to the address specified in the `source_address`
+  // field. This enables to specify multiple source addresses. Currently, only one extra
+  // address can be supported, and the extra address should have a different IP version
   // with the address in the `source_address` field. The address which has the same IP
   // version with the target host's address IP version will be used as bind address. If more
-  // than one additional address specified, only the first address matched IP version will be
+  // than one extra address specified, only the first address matched IP version will be
   // returned. If there is no same IP version address found, the address in the `source_address`
   // will be returned.
-  repeated SocketAddress additional_source_addresses = 4;
+  repeated ExtraSourceAddress extra_source_addresses = 5;
+
+  // Deprecated by
+  // :ref:`extra_source_addresses <envoy_v3_api_field_config.cluster.v3.UpstreamBindConfig.extra_source_addresses>`
```

Comment:
The doc test seems failed on the label https://dev.azure.com/cncf/envoy/_build/results?buildId=117819&view=logs&j=91e633a5-4907-5da1-5862-92a79fe2387a&t=f29e24e7-a093-5724-80bd-d79975d268cb&l=102

but I couldn't reproduce this error in my local.

is the label case sensitive? I saw the warning message using the low case of UpstreamBindConfig, ` envoy_v3_api_field_config.cluster.v3.upstreambindconfig.extra_source_addresses`. Or I missed something. 

@phlax do you know what happened here? thanks in advance!

Reply:
May be the correct label should be `envoy_v3_api_field_config.core.v3.BindConfig.extra_source_addresses`?

Reply:
oh, right, thanks!

this is seems strange https://github.com/envoyproxy/envoy/pull/23270/files#diff-7d01a4eae2ec35f22ebc30d24d2e08800e159a3f913efc9fb50f6de3c59e24a7R133, we have another message called UpstreamBindConfig

Reply:
see if that can be removed https://github.com/envoyproxy/envoy/pull/23323